### PR TITLE
Feature/pxweb2 90 label

### PR DESCRIPTION
**Pull request by MikaelNordberg**
Label component

### CODE REVIEW

File: libs/pxweb2-ui/src/lib/components/Typography/Label/Label.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+export function Label({
+  size = 'medium', 
+  align = 'start',
+  textcolor = 'default',
+  visuallyHidden = false,
+  children,
+  ...rest
+}: LabelProps) {
+  return (
+    <p
```
Comment (michaelpande) [inline libs/pxweb2-ui/src/lib/components/Typography/Label/Label.tsx:21]: This should probably be a span. A `<p>` tag is for paragraphs

Comment (MikaelNordberg) [reply libs/pxweb2-ui/src/lib/components/Typography/Label/Label.tsx]: @michaelpande 

Now is a label element with the possibility to set htmlFor:

>       <Label htmlFor="fname">First name:</Label><br />
>       <input type="text" id="fname" name="fname" defaultValue="John" /><br />
>

**Review by KentMossback — DISMISSED** [review]
Looks good

**Review by KentMossback — APPROVED** [review]
Still looks good

**Review by PerIngeVaaje — APPROVED** [review]
Looks OK

**Pull request merged**